app/services/alchemy/search/search_page.rb

```diff
@@ -6,6 +6,10 @@ module SearchPage
       def self.perform_search(params, ability: nil)
         search_results = Alchemy.search_class.search(params[:query], ability:)
         search_results = search_results&.page(params[:page])&.per(paginate_per) if paginate_per.present?
+
+        # order the documents by searchable_created_at and use the ranking order as second order argument
+        search_results.order_values.unshift("pg_search_documents.searchable_created_at DESC") if params[:sort] == "date"
```

Comment:
we should wrap the if condition in a block. otherwise it is very hard to find on small screens